### PR TITLE
Feature page

### PR DESCRIPTION
**Pull request by abeidahmed**
- Add header
- Add an about page
- Add a restaurant menu page with all the menus listed
- Add a contact page

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this project: everything looks good :+1: 
However, some technical changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Since this is a solo project, change `Authors` to `Author` in your Readme.
- [x] Necessary run common commands like build, development should be in package.json inside `scripts`. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
```JSON
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
- [x] Use of arrow functions (ES6 syntax) instead of the `function` keyword for your function declarations. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by rindrajosia — APPROVED** [review]
Hello! Great work implementing the requested changes!

Project Approved! :100: 

Cheers! :medal_military:

**Pull request merged**
